noxfile.py

```diff
@@ -270,49 +266,42 @@ def qa(session: nox.Session) -> None:
 @nox.session(python=False)
 def lint(session: nox.Session) -> None:
     """Run the linters."""
-    session.notify("flake8")
+    session.notify("ruff_check")
+    session.notify("ruff_format")
     session.notify("bandit")
 
-    # Black installs under Pypy but doesn't necessarily run (cf psf/black#2559).
-    if platform.python_implementation() == "CPython":
-        session.notify("black")
-    else:
-        session.log(
-            f"Skip black because of platform {platform.python_implementation()}."
-        )
-
 
 @nox.session
-def flake8(session: nox.Session) -> None:
-    """Run flake8."""
-    session.install("flake8", "flake8-print")
+def ruff_check(session: nox.Session) -> None:
+    """Run bandit, a code formatter and format checker."""
```

Comment:
Docstring incorrectly references "bandit"? Seems like after a couple rounds of changes, many docstrings have become out of date.